### ISSUE TITLE
Fix `color-function-notation` false positives for namespaced imports

### ISSUE THREAD

**Issue by PhilippeMorier** [stylelint/stylelint#6610]
### What steps are needed to reproduce the bug?

See [demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcIAkAjAhleACATgcxQApIAbCHAOlRLTAGsAaLABgoEYBKAbgB0A7JHgjRs+ItVqMW7bv36oMAJlEE0xCGUoT6TVp14Ch0ZblWFtUvbL4gGIAM4BPPgBc0ADwQOw9+7fAQfABmAJZ4XsD8WFg8IDgAriQw9rHYkXzR0bFoLgC0CUm5fBC58Xx0xQDufKlYLgkwDFGZsShk9EUlMAC2AA4ujrX18Y3NWQGanbkhfABuaCQhULkAFjDuQw1NGS0B3d0wrlM9-YMIdVtjMeDx9i4Q3bm9OBC9MDgDU90hvjN4ufMcLkgmUwC4QoFNiNtplrrAwDQcDkIXxcm0IB1irkoPFeoswDkYLkwLd7o9nq93uDklDRjtxvDEcjAmj2nQpji8SECS4iRS3h8QjTzgBtK7RYZ02HRdLS3ZhYo4GC1MX0uXXSB8ewwEng2ZEzn4wn2XKVEIuFbYkJBILvQ55eYkEYpEDizIAXTdAF8ru6YbtGWgkeCWejMSV7CtyBa0HxlvyqY5chB9TgcEthdhJf7xkFAnkgmhviQk1jDdzCUUi5mLtCrrE80dC8XSyVvr8+P88Id3tzgUWQiXcnQYI5KuQoLSc9cQXwwSjiQswFMyvZerQYMtKcHyFP6yBZ-OWYs+DAg7k8EioEKjljilq3HGg8tr0qjzVztn9yPHEEkQdsR1JkQ1RLEQj6aNYxcPc1ViA5rzQYEzxceIlSrACsTKCoIGqGDYViPhq2WS8llyIMzxNMC5gWJY8N2MtkjAQ5r07XI1x1a1uXNM4s0uWCQDLXEjV5Mi8nA3po3yRIay-fjBK5HkiW1JIwXIF1eLrOSuj6T57AgVCmLo8YsXWepEO1b5SECdTayleiShmR1SKgfS2iUmhI2JB4DlcGzZPwgSHOoxZ4wgexzQXHJpggj4pKSIzrjAtMYDwRJz0qFZzWSddDM-PiAoTD5W1Kcoqg-DS7PGZSdXuIFem1eIXOJDzKJKLCyoS2JqtUuqGqamAkh8vJMNKnDyts6cuoGmryFyAY3hXUbcNFN1-OlWV1ViBVyGVFa1TlWISTuB5cmmoaXTdaJPX2rAfTVP19zudNWLvGBKhPXaKsmkAynNRbsOWr79wWXoVkQx0Rk6NwQJVN1YgFJjXDQbtYmnGVLuudYmP6AAFF4BWpGzVXVXZXlobjYgx665Tu2EHv4qKCiJE5Pg+tEYDzJVYZu7ISEqNBHBdNGsA2g6QCxmB+m5knxjDOgkl8Migl5IF7GrdCiTlhWhYx3MQhwO4imSXlJ1dG6ruF8Ztq5vaZeuNBlfeLz9ntSnzawanpVpj1vsZ6SNcXbValiMhKneVHgbyJmA7XDclZV4OQAWfnBbYv5Cg+iOGaj-2sX1ONZueDmQg2PLNICv3Cgs7kNBZU93pmIkHYT85eZTnX+Ll5qwr+NEkSY06dKTNmUA5nbE9PVMs4CruER71iUH7ol67Z5vw9bpO+YFjuZ7ZbvwoXpejYb092c5z77a31PukScFckz2x91nsgD-+Re49jgfR-PxPk+3tPOwZ0btPXYXdKR8F7u-AeK9G7x3XtgNu-8b4kDvg-b6YC3gQMPh-HKTdHY4F-lfE0r8gGnhAbLPe4DIFH0-prMeNsEGb3buQjUtcgSHhAlDZk404LQHeB+b6VkgRrHcIHC+IccLr0EWw1Y6x76HDwBaROkZowsMOt5e0g9Tj31gd-cettpSIMFiw9G7tYgSylgYkmDZ9aG1PHcTcbsSb02sSAa2F9iYyy6gMJIJ48iQH2LGKAF13ae1hN7C2+4AlDVNJlXktDphagzIQ5hj9+JHTJE8fGiYtGs10fQjxcMmHbxMSLDG5j3DY2glYzaSd8HElJCdQqAxSkBVCAbYaxtHFm2cZba47jpZeLqSrZ2Q1WnyiSbAABeBSGazZE49UYTMgRI9oIxpjx4IhEQuuFwKtxqeICgAPUICKRCAAvd0pzchnOYLkAAnO6AAVBwQguQrk3Pue6AA1BwR5SBSmi0MSAA4vhkbiJAAAUXcG8MEm4sAZIeFgTZaAsAAEcRg4EcFgQiBw6gQCwKPLAI50AoDEQs8Jvo1nHXJNkoqTwch7MGUck55zLnnNuQ855rz3kcu+b8-5aS5SAoCiCtWKMN5QphSbeF6ysDNKxTimAeKCVKuJWgUlBIg49K9pS-cgZgwLnQKxBJa8CEb0nlIvVQEgzcLRLGf4CS9EMMvqk76+rbVd2rlZUCb1V74JSUg2+IQdFkMFQGa1BrQx7y9bXVEJr-Ubz-qnEhRI0FWoRDazhnqeg1zIHG3BZ99GMItWat1EaPXRvTqm2B3QPCAQzZGrUtQ2BlobRWjE7IzKDl7jGvNAbjFhoZOWrNe8sRKhxE+I4ea8BgzjFk7chNOogHdZw71x8-Ut0YUmk0yDUHAMHXCYdC413xs3S6-+KaQ27VbcBY9sa2IFqdeCktajl1HpZCzYe+Sf41PGNugF5TxaVMltU7ABzammvrbeyEB6xbtLsV002VM+lbTwIqQp7s-31Oia7WDQKZjhSmZeke8ztVyiWdEFZLjdiNgLAOIcircjoogPE-tJoMp2nyDqDRcZunfQ4XewJ67YGmrY7kXdwa038QEyyaJ5kC2icTUQ6ZszX0ydRHJh9ccn0TxgFPA9DZQScNraIz9V6bLMH40ZhcjHNXgtDpa6T1mWTriVK4NY2pKK+tgQR5JSn27iaDVetTznUSuftB55IWmB6+dgLp1MKnq2hqs3OThqESBMfiCxmsRid40dC7ErKp74HnoHd9FYkNYCXhgMvFj3Df6AOvfuGYsAkYw3OIob6P4-zVkourHZjLf3XGOe8tl1zeVcreeyz5Py-kAbMcC5IYrwWSpqnC7r-4lWKuVYStVGq0Bau9Lq-iSROwWmue8EoI1zRLpM7kr99jm3fTu2zU7iiVjNsUJZ-cyKkI5FQkSE9CmE1bqIa+37tp-toSB9pgp8XHMis3Fsv7KE0K2YO-ZyRpaftI8QpD1Hy91Z52YoXJUoRS5A34hD5CAP6VuYtMkKLCTYvPr0wj3Y1OodEiRJ2Ik24cizWK2a0HrqccIRR7Tnn3ZkwCgF6rR9cP-MlIM4t8X6L3hfsNppmBp9FMi8DSgyT+7nu48yxrnRWvvLybjnr0rxCq3BZV799XmKLf+Kt9FuhP7i1s+x1pe7wJbF5AfpTgKJkNCFfibgpdWJ2yv1OnOiAQQ2L6RwNA7zobQ-0XiN0UeQIkgYF7mcy7bHX18Bz3n8Tdai5gB+CiWoAAWb65fc9OyxF2k8-xi8vD8vlXY8qyUbwc37gqtLPjE4LnVMnJcl3RzMyPRXYGinbsC4bx3wthW7AsaBrA4G4NB6Ng4pDoSUNuLQ0W3fut7bYZ49UqmR37rfW6rVUS9QQgoHiCJKBI4XDEILSz+HEfXYZ-WaBldMD-ESfnF-IXQA19EAoEMA9-T-PnWXaAhXb3RKX3OA6aHqV-cA5AzLbLGyXLbAlSF-DNRWAbfhJlXYEbVlHlT5SbBgh5WbAVDfQDUVMFROVbWFKAeFFqLAeAnbVVGAElQfB-OmJ-HA8gh4FAGYOXT3OBYXO3UgmaIEAJOQwiNA2HDAkglXeAwCewRGKAKCZ2TQhQu8FkWhJdAw0iKgnAfZIpOg65MbD5TlF5KbcbGbflebVxTg8VRhHg6VJYQQ6Q8gYQolUQ9VcQm6KjKQsg2aRYS3ITHXPBM9PQ+ItQt3UZa3L+RfTA-TTI3Au7efRuCzIol-CfFyKfYuCnCaR6MI3qGARqEoCgk0OzROYfVQ3A+qZopqNounCLRnP-OOAA81LA-Qxop4PqLoQaTRDoofLHbol-Xolo06OYo4NdYoaGevDeFyeINyZY2aeaQHDHTopY-QqMD4WdeMMfYqcdMoEwo4CGGSPvKqN-ViZjVjPY1yeKFXX6fxM4xYsOIA8YF44cUcccHAZYBYxhLolXcEpI93FIjPNIkrXLVfPdZLfcREn4ZE2tRQ23DE4jY3HEhYSGJEnIxQnTcYwoskp0atQ2EooeczWob7ficEqo0nWo2kZoO6L0EAL0IAA) showing the issue.

```css
$bad: rgb(color.$black, 0.1);
$bad2: rgba(color.$black, 0.1);
```

### What Stylelint configuration is needed to reproduce the bug?

```json
{
  "ignoreFiles": ["**/*"],
  "overrides": [
    {
      "files": ["**/*.css"],
      "extends": ["stylelint-config-standard", "stylelint-config-prettier"],
      "rules": {}
    },
    {
      "files": ["**/*.scss"],
      "extends": ["stylelint-config-standard-scss", "stylelint-config-prettier"],
      "rules": {}
    }
  ],
  "rules": {}
}
```

### How did you run Stylelint?

Within `nx` workspace & `nx-stylelint` plugin from @Phillip9587: `nx stylelint my-app`

### Which version of Stylelint are you using?

`14.16.1`

### What did you expect to happen?

No error/warning showing since syntax is correct.

### What actually happened?

Error is reported:
> Expected modern color-function notation ([color-function-notation](https://stylelint.io/user-guide/rules/color-function-notation))

### Proposal to fix the bug

Maybe related to https://github.com/stylelint/stylelint/issues/5671.

**Comment by ybiquitous** [comment]
@PhilippeMorier Thanks for the report with the minimal reproduction.

I've labeled the issue as **ready to implement**. Please consider [contributing](https://stylelint.io/contributing) if you have time.

There are [steps on how to fix a bug in a rule](https://stylelint.io/developer-guide/rules#fix-a-bug-in-a-rule) in the Developer guide.
